duckietown_rl/ddpg.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, action_dim, max_action):
         super(ActorCNN, self).__init__()
 
         # ONLY TRU IN CASE OF DUCKIETOWN:
-        flat_size = 32 * 9 * 14
+        flat_size = 32 * 2 * 2  # FIXME If large change to flat_size = 32 * 9 * 14
```

Comment:
For clarity I would suggest you change this to:
`# FIXME If using larger 120x160 image input change to flat_size = 32 * 9 * 14`